### PR TITLE
Apply pep8 standard to PySvn

### PR DESCRIPTION
**Pull request by rnt**
Apply pep8 standard to PySvn

### CODE REVIEW

File: svn/common.py [hunk 1]
```diff
@@ -411,19 +416,22 @@ def diff_summary(self, old, new, rel_path=None):
         root = xml.etree.ElementTree.fromstring(result)
         diff = []
         for element in root.findall('paths/path'):
-            diff.append({'path': element.text, 'item': element.attrib['item'], 'kind': element.attrib['kind']})
+            diff.append({'path': element.text, 'item': element.attrib[
```
Comment (dsoprea) [inline svn/common.py]: I prefer:

```
diff.append({
    'path': ..,
    'item': ..,
})
```

File: svn/common.py [hunk 2]
```diff
@@ -390,14 +394,15 @@ def list_recursive(self, rel_path=None, yield_dirs=False,
                 if entry['is_directory'] is False or yield_dirs is True:
                     current_rel_path_phrase = \
                         current_rel_path \
-                            if current_rel_path is not None \
-                            else ''
+                        if current_rel_path is not None \
```
Comment (dsoprea) [inline svn/common.py]: Similar to above.

File: svn/common.py [hunk 3]
```diff
@@ -171,56 +175,57 @@ def properties(self, rel_path=None):
...
         # now query the content of each propery
         property_dict = {}
 
         for property_name in property_names:
             result = self.run_command(
-                        'propget', 
-                        ['--xml', '{0}'.format(property_name), full_url_or_path, ], 
-                        combine=True)
+                'propget',
+                ['--xml', '{0}'.format(property_name),
```
Comment (dsoprea) [inline svn/common.py]: Please change `'{0}'.format(property_name)` to `property_name` while you're at it.

File: svn/common.py [hunk 4]
```diff
@@ -118,27 +119,27 @@ def info(self, rel_path=None):
             'repository/uuid': root.find('entry/repository/uuid').text,
 
             'wc-info/wcroot-abspath': wcroot_abspath.text \
-                                        if wcroot_abspath is not None and \
-                                           len(wcroot_abspath.text) \
-                                        else None,
+            if wcroot_abspath is not None and \
```
Comment (dsoprea) [inline svn/common.py]: This is a lot harder to read without the indentation.

File: svn/common.py [hunk 5]
```diff
@@ -102,13 +103,13 @@ def info(self, rel_path=None):
...
-                                   len(relative_url.text) \
-                                else None,
+            'relative_url': relative_url.text
+            if relative_url is not None and
+            len(relative_url.text)
+            else None,
 
-# TODO(dustin): These are just for backwards-compatibility. Use the ones added 
-#               below.
+            # TODO(dustin): These are just for backwards-compatibility.
```
Comment (dsoprea) [inline svn/common.py]: Please remove the indentation. Keep the TODOs all of the way to the left so that they stand-out.

File: svn/common.py [hunk 6]
```diff
@@ -23,9 +24,9 @@ def __init__(self, url_or_path, type_, *args, **kwargs):
 
         self.__type = type_
 
-    def run_command(self, subcommand, args, success_code=0, 
+    def run_command(self, subcommand, args, success_code=0,
                     return_stderr=False, combine=False, return_binary=False):
-# TODO(dustin): return_stderr is no longer implemented.
+        # TODO(dustin): return_stderr is no longer implemented.
```
Comment (dsoprea) [inline svn/common.py]: Remove indentation.

**Comment by dsoprea** [comment]
Thank you. I made a few notes. Also, please merge. This seems to be in conflict.

File: svn/common.py (hunk 1 above)
Comment (rnt) [inline svn/common.py]: Yeap! Me too!
autopep8 not thinks alike.

File: svn/common.py (hunk 5 above)
Comment (rnt) [inline svn/common.py]: @dsoprea but this TODO is not a TODO... Is like a warning. :cry: 

Maybe delete this comment and change by a message in README.md... and in a mayor release... drop the backwards compatibility.

**Comment by dsoprea** [comment]
Please resolve the conflicts because of another PR.

**Comment by rnt** [comment]
@dsoprea ready!

**Pull request merged**